integration/integration.dhall

```diff
@@ -1,4 +1,4 @@
-let conf = ./spago.dhall
+let conf = ../spago.dhall
```

Comment:
I think it's a little nicer to keep these files in the directories where they are used.